BrowserKit/Sources/WebEngine/WKWebview/Internal/InternalUtil.swift

```diff
@@ -6,6 +6,7 @@ import Foundation
 
 /// Used to setup internal scheme handlers
 struct InternalUtil {
+    @MainActor
```

Comment:
This was needed since `WKInternalSchemeHandler.responders[path] = responder` crashes if it's not on the main thread